packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -293,14 +308,27 @@ module.exports = function(log, config, oauthdb) {
   Mailer.prototype.localize = function(message) {
     const translator = this.translator(message.acceptLanguage);
 
-    const localized = this.templates[message.template](
-      extend(
+    let localized;
+    if (message.layout === 'subscription') {
```

Comment:
For now there are two different templating systems in play, but when I fix #1779 this condition will disappear.